WORKSPACE

```diff
@@ -268,6 +268,24 @@ rbe_autoconfig(
     use_checked_in_confs = "False",
 )
 
+rbe_autoconfig(
+    name = "rbe_autoconf_generate_windows",
+    bazel_version = "3.1.0",
+    create_testdata = True,
+    use_checked_in_confs = "False",
+    digest = "sha256:4fa38fa8c1070850b098d8c03334a86774fffa87004a0ce601d87a883745f9f0",
+    registry = "gcr.io",
+    repository = "envoy-ci/envoy-build-windows",
```

Comment:
Currently we have just hard coded this image from the Envoy project here as a way to get started (built from here: https://github.com/envoyproxy/envoy-build-tools/tree/master/build_container), will we be able to build an image specific to this project that can be hosted in gcr? I do see in the experimental checked in configs for Windows we have: https://github.com/bazelbuild/bazel-toolchains/blob/e7cd66eccd2532c7601fbac3c6f93088cb5b4153/configs/experimental/windows/0.1.0/BUILD#L38 and the checked in ubuntu ones are hosted in the google project here: https://github.com/bazelbuild/bazel-toolchains/blob/e7cd66eccd2532c7601fbac3c6f93088cb5b4153/configs/ubuntu16_04_clang/releases.yaml#L4